src/main/java/ch/cern/eam/wshub/core/services/workorders/impl/ChecklistServiceImpl.java

```diff
@@ -108,9 +108,13 @@ private void getResponsibilityDescriptions(InforContext context, WorkOrderActivi
 		String reviewerQualification = workOrderActivityCheckList.getReviewerQualification();
 		String performer1Qualification = workOrderActivityCheckList.getPerformer1Qualification();
 		String performer2Qualification = workOrderActivityCheckList.getPerformer2Qualification();
-		List<String> qualifications = new ArrayList<>();
-		for(UserQualification qualification : workOrderActivityCheckList.getUserQualifications())
-			qualifications.add(qualification.getUserDefinedCode());
+		List<String> qualifications = null;
+
+		if(workOrderActivityCheckList.getUserQualifications() != null){
```

Comment:
Good thinking to check for null. If we initialize `List<String> qualifications` as `new ArrayList<>();` we also will not need null checks below, since an empty list never contains anything.